autoscheduler/frontend/src/tests/MeetingCard.test.tsx

```diff
@@ -33,39 +36,23 @@ const testMeeting = new Meeting({
   section: testSection,
 });
 
-test('accepts meeting and color as props', () => {
-  const { container } = render(
-    <MeetingCard
-      meeting={testMeeting}
-      bgColor="#500000"
-      firstHour={8}
-      lastHour={21}
-    />,
-  );
-  expect(container).toBeTruthy();
-});
+let store: Store = null;
+
+beforeAll(() => { store = createStore(autoSchedulerReducer); });
 
-test('displays subject and course number', () => {
-  const { getByText } = render(
-    <MeetingCard
-      meeting={testMeeting}
-      bgColor="#500000"
-      firstHour={8}
-      lastHour={21}
-    />,
+test('accepts meeting and color as props, displays subject, course number, and meeting type', () => {
+  const { container, getByText } = render(
+    <Provider store={store}>
+      <MeetingCard
+        meeting={testMeeting}
+        bgColor="#500000"
+        firstHour={8}
+        lastHour={21}
+      />
+    </Provider>,
   );
+  expect(container).toBeTruthy();
```

Comment:
I'm leaning towards leave it, just because with the new `ScheduleCard`, just to test the basic functionality of the `MeetingCard` and to keep up the code coverage of the tests.